### ISSUE TITLE
Manual sorting of table rows

### ISSUE THREAD

**Issue by 57even** [aslagle/reactive-table#254]
I thought I could change

    {{#each sortedRows}}
      ..
    {{/each}}

to

    {{#sortable items=sortedRows}}
      ..
    {{/sortable}}

in the template in order to use https://github.com/RubaXa/Sortable/tree/master/meteor with `reactive-table`. The sorting works great but it wont save the ordering because it seems that `reactive-table` is changing the collection to a list, making collection-related operations such as `.update(modifier)` unaccessible. Is this correct? Are there any chances for implementing manual sorting in `reactive-table`?

**Comment by aslagle** [comment]
Yes, it's necessary to use a list in order to sort with fn instead of just mongo's sort. I don't know anything about the sortable package, so I'd have to spend some time looking into whether it would be possible to integrate it or not. I wouldn't want to add a dependency on sortable to reactive-table, but I'd like to make it possible to use custom templates for different parts of the table which might enable it.
